static/src/stylesheets/layout/garnett-header/_pillar-link.scss

```diff
@@ -1,18 +1,18 @@
 .pillar-link {
-    font-family: 'Guardian Egyptian Web', 'Guardian Text Egyptian Web', Georgia, serif;
+    font-family: 'Guardian Egyptian Web', 'Guardian Text Egyptian Web', TimesNewRoman, serif;
```

Comment:
ooh! Times New Roman!

Reply:
Oh sorry - should have left a note there. Georgia is so wide at bold that it breaks the layout, so rather than build a no js alternative I switched the fallback font.